linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_overflow_action/ansible/shared.yml

```diff
@@ -2,5 +2,7 @@
 - name: "{{{ rule_title }}}"
   lineinfile:
     path: /etc/audit/auditd.conf
-    regexp: '^overflow_action'
+    regexp: '^.*overflow_action'
```

Comment:
without this, you would miss on a line that is commented or has a space with comment